guides/common/modules/proc_upgrading-packages-on-a-host.adoc

```diff
@@ -1,14 +1,18 @@
 [id="upgrading-packages-on-a-host_{context}"]
 = Upgrading Packages on a Host
 
-You can upgrade packages on a host in bulk in the {ProjectWebUI}.
+The package upgrade process takes place via Katello agent or remote execution, depending on your configuration.
+To upgrade the packages on a host, follow the procedure below.
 
 .Procedure
 . In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
 . Click the name of the host you want to modify.
 . Click the *Content* tab, then click the *Packages* tab.
-. Select *Upgradable* from the *Status* list.
-. Select upgrade version from the dropdown menu in *Upgradable to* column where applicable.
-. Select the packages you want to upgrade.
-. Click the *Upgrade* button.
+. Select *Upgradable* from the *Status* dropdown menu.
+. Select the packages from list and click *Upgrade*.
+If a package has more than one available upgrade version, only the latest upgradable version is displayed.
+
+You can also run a remote execution job immediately by selecting *Upgrade via remote execution* from the *Upgrade* dropdown menu.
```

Comment:
We can also add the link to `Configuring and Setting Up the Remote Execution`, same as the one described in Installing Package part (last lines).